packages/react-native-sdk/src/components/ActiveCall.tsx

```diff
@@ -21,6 +22,10 @@ export interface ActiveCallProps {
    * Handler called when the participants info button is pressed in the active call screen.
    */
   onOpenCallParticipantsInfoView: () => void;
+  /**
+   * When true, the default participant list sorter will be disabled.
+   */
+  disableDefaultSorter?: boolean;
```

Comment:
Rather than `disableDefaultSorter`, can we name the prop as `enableDefaultSorter` and keep the default value as `true`? This will make the code a bit more readable, IMO. Feel free to ignore my suggestion if it doesn't make sense.

Reply:
This is sort of a normal RN way... 

example: https://reactnative.dev/docs/pressable#disabled 

let me know if it makes sense

Reply:
It's either disable or enable, both are fine. Yet I feel like It makes sense to me to do disable a default functionality over enabling it, indeed @santhoshvai